_studio/shared/umc/codec/h265_dec/src/umc_h265_heap.cpp

```diff
@@ -343,9 +343,6 @@ uint8_t * CumulativeArraysAllocation(int n, int align, ...)
     int cumulativeSize = 0;
     for (int i = 0; i < n; i++)
     {
-        void * ptr = va_arg(args, void *);
-        ptr; // just skip it
-
         int currSize = va_arg(args, int);
```

Comment:
But actually, you don't need to fix the above since this whole function is simply not used:
```
fgrep -rsn CumulativeArraysAllocation _studio/
_studio/shared/umc/codec/h265_dec/include/umc_h265_heap.h:36:uint8_t * CumulativeArraysAllocation(int n, int align, ...);
_studio/shared/umc/codec/h265_dec/include/umc_h265_heap.h:37:// Free memory allocated by CumulativeArraysAllocation
_studio/shared/umc/codec/h265_dec/src/umc_h265_heap.cpp:338:uint8_t * CumulativeArraysAllocation(int n, int align, ...)
_studio/shared/umc/codec/h265_dec/src/umc_h265_heap.cpp:374:// Free memory allocated by CumulativeArraysAllocation
```
Just remove it as a whole.